File: src/main/java/um/tesoreria/report/domain/dto/ArancelTipoDto.java
package um.tesoreria.report.domain.dto;

import lombok.Data;

@Data
public class ArancelTipoDto {
    private Integer arancelTipoId;
    private String descripcion = "";
    private Byte medioArancel = 0;
    private Integer arancelTipoIdCompleto;
}

File: src/main/java/um/tesoreria/report/kotlin/dto/tesoreria/core/ArancelTipoDto.kt
package um.tesoreria.report.kotlin.dto.tesoreria.core

data class ArancelTipoDto(

    var arancelTipoId: Int? = null,
    var descripcion: String = "",
    var medioArancel: Byte = 0,
    var arancelTipoIdCompleto: Int? = null

)
